community-manager-self-care.adoc

```diff
@@ -80,47 +80,47 @@ Set realistic expectations on deliverables and communicate frequently within the
-Our passion and attention is directed in a constructive and purposeful manner, this being especially important after when a mistake is made.
+The ultimate aim is to be "under control," both with your emotions and your workload.
+Here, our passion and attention are directed in a constructive and purposeful manner—a particularly important place to be in when we've made a mistake.
 
 The phrase "bring your whole self to work" is a good expression of allowing yourself this forgiveness.
-We feel we can admit when we don't know something, the right decision or simply acknowledging when we've made a mistake.
-But it also can be applied to recognizing and accepting that we can have good and bad days.
+We should feel like we can admit when we don't know something, haven't made the correct decision, or have made a mistake.
+And we should also recognize and accept that we can have good days and bad ones.
 
-Being honest with ourselves is as important as being honest with the community.
-Needing to admit that we may not be at our best and detracting yourself from unintentionally escalating  difficult situations into becoming worse situations would be the best choice to make at those times for the community.
+Being honest with ourselves is as important as being honest with our communities.
+Needing to admit that we may not be at our best and detracting yourself from unintentionally escalating difficult situations into becoming worse situations would be the best choice to make at those times for the community.
 
-This emotional labor can be extremely exhausting to maintain.
-We must acknowledge and accept that being perfect is unattainable, and more importantly, not required to be the best leader.
-What is important is that people can relate to your human side.
+Maintaining this kind of emotional labor can be incredibly exhausting.
+We must acknowledge and accept that being perfect is unattainable, and more importantly, not a requirement for being a great leader.
+What _is_ important is that people can relate to your human side.
 
-People gravitate to those that they share a kinship with and being able to identify this is one of the characteristics of a good community manager.
-If your members see that you have qualities that they can relate to, it helps them to more easily empathize with you.
-Funnily, we usually try to highlight the significance of our empathy for our members or team, but it's equally important that our members demonstrate compassion, and gratitude to us too.
+People gravitate to others with whom they share a kinship, and being able to identify this feeling of kinship is one hallmark of an effective community manager.
+If your members see that you possess qualities they can relate to, they can more easily empathize with you.
+Funnily enough, however, we usually try to highlight the significance of our empathy for our members or team, but it's equally important that our members demonstrate compassion, and gratitude to us too.
```

Comment:
Similar comment to line 109: I think the sentence could use some editing, but I'm not quite sure of your intent with it, so I'm not sure how best to help.

Reply:
(similarly to the other review) I have made amendments to hopefully make it a bit more readable. Let me know if it requires further clarification!

Reply:
Great! Thanks.